18-interrupts/kernel/util.c

```diff
@@ -26,5 +26,15 @@ void int_to_ascii(int n, char str[]) {
     if (sign < 0) str[i++] = '-';
     str[i] = '\0';
 
-    /* TODO: implement "reverse" */
+    // reverse
+    char aux_char = 0;
```

Comment:
Another solution could be implement stdlibc function `void strrev(char *str)` and use it here to reverse string.